### PR TITLE
Config with default template

### PR DESCRIPTION
**Pull request by gforcada**
_Remember to use `[ci-skip]` on the merge commit message to avoid spamming Jenkins_ ✨ ♻️

The last commit, that fixes the dependencies, is needed in order to fix https://github.com/plone/plone.portlets/pull/12

### CODE REVIEW

**Comment by gforcada** [comment]
@mauritsvanrees you made https://github.com/plone/meta/pull/113 which was a great idea ✨ 💯 

Unfortunately in this specific repository, where we still (?) want to test with `tox-gh-actions` we can not use plone/meta#113 trick 😕 

I would say that we can remove the `.github/workflows/tests.yml` 🍀

File: tox.ini [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-          pypy3,
+envlist =
+    lint
+    test
+    dependencies
+    py38,
+    py39,
+    py310,
+    py311,
+    pypy,
```
Comment (mauritsvanrees) [inline tox.ini:15]: `pypy` could actually be removed: this would be for PyPy2, which we do not support.
`pypy3` should stay though.

**Comment by mauritsvanrees** [comment]
> @mauritsvanrees you made [plone/meta#113](https://github.com/plone/meta/pull/113) which was a great idea ✨ 💯
> 
> Unfortunately in this specific repository, where we still (?) want to test with `tox-gh-actions` we can not use [plone/meta#113](https://github.com/plone/meta/pull/113) trick 😕
> 
> I would say that we can remove the `.github/workflows/tests.yml` 🍀

```
$ tox -e pypy3
pypy3: commands[0]> echo 'Unrecognized environment name pypy3'
```

I see... :-/
So this package is a special case. But you know what they say about special cases:

```
>>> import this
The Zen of Python, by Tim Peters
...
Special cases aren't special enough to break the rules.
```

It would be nice if we could support PyPy3 for this package.  All its `install_requires` support this.  But I doubt anyone uses this outside of Plone.
So for me it is fine to drop the special cases here, so mostly drop support for PyPy3, and make a new major release.  We can always see if we want to revert in case someone complains.

**Comment by gforcada** [comment]
A middle ground solution, would be to add a specific test for pypy3... Zope is meant to work on it, and so should Plone, or? 🤔

**Pull request merged**